pkg/resolvers/aggregateResolver_test.go

```diff
@@ -1,11 +1,12 @@
 package resolvers
 
+// test generated with:
+// mockgen --build_flags=--mod=mod -package resolvers -destination ./mockAuthRuleResolver_test.go "k8s.io/kubernetes/pkg/registry/rbac/validation" AuthorizationRuleResolver
```

Comment:
What's the motivation of moving this out of `mocks` or `fakes`. It seems counter to where we normally put generated mocks. Also, I don't think it should be named `_test` since there are no tests in the generated file